src/js/tabs/setup.js

```diff
@@ -418,7 +429,7 @@ setup.initialize = function (callback) {
                 const buildOptions = `<span class="buildInfoBtn" title="${i18n.getMessage(
                     "initialSetupInfoBuildOptionList",
                 )}">
-                                      <a class="buildOptions" href=#"><strong>${i18n.getMessage(
```

Comment:
Pure consistency change, this matches the formatting 7 lines above. I think prettier should have made this itself.